pages/work-ad.js

```diff
@@ -37,10 +37,25 @@ export default class WorkAd extends React.Component {
               <h1>{ad.title}</h1>
               <p>{ad.subtitle}</p>
             </div>
-            <div
-              className="hero-media"
-              style={{ backgroundImage: `url(${mkUrl(ad.imageUrl)})` }}
-            />
+            <picture className="hero-media">
+              <source
+                media="(max-width: 400px)"
+                srcSet={mkUrl(ad.imageUrl)
+                  .width(800)
+                  .url()}
+              />
+              <source
+                media="(max-width: 799px)"
+                srcSet={mkUrl(ad.imageUrl)
+                  .width(800)
+                  .url()}
+              />
+              <source
+                media="(min-width: 800px)"
+                srcSet={mkUrl(ad.imageUrl).url()}
+              />
+              <img className="hero-image" src={mkUrl(ad.imageUrl)} />
```

Comment:
alt text on image?

Reply:
well, yes. `alt=""` and role=`presentation`.